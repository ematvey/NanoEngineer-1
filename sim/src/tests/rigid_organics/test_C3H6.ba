LENGTH 0 1 1.50778413574
LENGTH 0 2 1.50791843281
LENGTH 0 3 1.08628817539
LENGTH 0 4 1.08709015266
LENGTH 1 2 1.50846975442
LENGTH 1 5 1.08667198363
LENGTH 1 6 1.08630474546
LENGTH 2 7 1.08603406945
LENGTH 2 8 1.08675572232
ANGLE 0 1 2 59.9908479338
ANGLE 0 1 5 118.25950599
ANGLE 0 1 6 118.165748245
ANGLE 0 2 1 59.9820140075
ANGLE 0 2 7 118.204401309
ANGLE 0 2 8 118.202128
ANGLE 1 0 2 60.0271380588
ANGLE 1 0 3 118.245249028
ANGLE 1 0 4 118.176891441
ANGLE 1 2 7 118.218292943
ANGLE 1 2 8 118.157594759
ANGLE 2 0 3 118.197293135
ANGLE 2 0 4 118.213498553
ANGLE 2 1 5 118.245429296
ANGLE 2 1 6 118.130988192
ANGLE 3 0 4 113.830978349
ANGLE 5 1 6 113.863575642
ANGLE 7 2 8 113.878609915
