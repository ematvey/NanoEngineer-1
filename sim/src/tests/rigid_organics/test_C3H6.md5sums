stdout A14FE025D15C6AAD6534E60EACD7E89F
stderr 160E6504290AEB20212DA51B283BAFE4
