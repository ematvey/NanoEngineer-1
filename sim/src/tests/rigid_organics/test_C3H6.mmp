mmpformat 050502 required; 050706 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (1.000000, 0.000000, 0.000000, 0.000000) (5.062915) (-0.000000, -0.000000, -0.000000) (1.000000)
egroup (View Data)
group (C3H6)
info opengroup open = True
mol (C3H6.pdb) def
atom 1 (6) (-813, 570, -65) def
atom 2 (6) (247, 204, -194) def
bond1 1
atom 3 (6) (-317, -1071, 63) def
bond1 1 2
atom 4 (1) (-1339, 1130, -828) def
bond1 1
atom 5 (1) (-1221, 1053, 597) def
bond1 1
atom 6 (1) (1361, 796, 1109) def
bond1 2
atom 7 (1) (1056, 1008, -767) def
bond1 2
atom 8 (1) (-464, -1560, -975) def
bond1 3
atom 9 (1) (-62, -1022, 1066) def
bond1 3
egroup (C3H6)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part C3H6


