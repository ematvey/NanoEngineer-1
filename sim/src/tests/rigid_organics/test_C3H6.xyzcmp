9
C3H6.mmp
C -0.800000 0.808000 0.000000
C 0.673000 0.486000 0.000000
C -0.343000 -0.629000 0.000000
H -1.239000 1.207000 -0.910000
H -1.238000 1.208000 0.911000
H 1.239000 0.666000 0.910000
H 1.237000 0.665000 -0.911000
H -0.470000 -1.208000 -0.910000
H -0.469000 -1.208000 0.911000
