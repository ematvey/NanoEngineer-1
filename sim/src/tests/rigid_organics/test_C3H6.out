======= test_C3H6.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 0
== structure comparison ==
error reading atom count from xyz file: test_C3H6.xyz
could not read comparison positions file
